rs/config/src/subnet_config.rs

```diff
@@ -343,35 +337,7 @@ impl SchedulerConfig {
     }
 
     pub fn verified_application_subnet() -> Self {
-        // When the `install_code` instruction limit on application subnets is
-        // also increased to 300B, then this line can be removed.
-        let max_instructions_per_install_code = NumInstructions::from(300 * B);
-        Self {
-            scheduler_cores: NUMBER_OF_EXECUTION_THREADS,
-            max_paused_executions: MAX_PAUSED_EXECUTIONS,
-            subnet_heap_delta_capacity: SUBNET_HEAP_DELTA_CAPACITY,
-            heap_delta_initial_reserve: HEAP_DELTA_INITIAL_RESERVE,
-            max_instructions_per_round: MAX_INSTRUCTIONS_PER_ROUND,
-            max_instructions_per_message: MAX_INSTRUCTIONS_PER_MESSAGE,
-            max_instructions_per_message_without_dts: MAX_INSTRUCTIONS_PER_MESSAGE_WITHOUT_DTS,
-            max_instructions_per_slice: MAX_INSTRUCTIONS_PER_SLICE,
-            instruction_overhead_per_execution: INSTRUCTION_OVERHEAD_PER_EXECUTION,
-            instruction_overhead_per_canister: INSTRUCTION_OVERHEAD_PER_CANISTER,
-            instruction_overhead_per_canister_for_finalization:
-                INSTRUCTION_OVERHEAD_PER_CANISTER_FOR_FINALIZATION,
-            max_instructions_per_install_code,
-            max_instructions_per_install_code_slice: MAX_INSTRUCTIONS_PER_INSTALL_CODE_SLICE,
-            max_heap_delta_per_iteration: MAX_HEAP_DELTA_PER_ITERATION,
-            max_message_duration_before_warn_in_seconds:
-                MAX_MESSAGE_DURATION_BEFORE_WARN_IN_SECONDS,
-            heap_delta_rate_limit: NumBytes::from(75 * 1024 * 1024),
-            install_code_rate_limit: MAX_INSTRUCTIONS_PER_SLICE,
-            dirty_page_overhead: DEFAULT_DIRTY_PAGE_OVERHEAD,
-            accumulated_priority_reset_interval: ACCUMULATED_PRIORITY_RESET_INTERVAL,
-            upload_wasm_chunk_instructions: DEFAULT_UPLOAD_CHUNK_INSTRUCTIONS,
-            canister_snapshot_baseline_instructions:
-                DEFAULT_CANISTERS_SNAPSHOT_BASELINE_INSTRUCTIONS,
-        }
```

Comment:
The diff seems big but essentially after changing the `install_code` limit, the two configs are exactly the same. In fact, I contemplated whether we should simply remove `verified_application_subnet` completely and just use `application_subnet` but I wasn't sure.